serialize.go

```diff
@@ -21,12 +27,15 @@ func SaveToFile(j *Jar, path string) error {
 	return j.WriteTo(f)
```

Comment:
yes. there are quite a few problems with this package as it is - it is deliberately naive, just enough to be going on with for the time being.